tests/standardise/test_standardise.py

```diff
@@ -443,3 +452,24 @@ def test_standardise_footprint_different_chunking_schemes(caplog):
 
     # Check that the chunking scheme is what was specified with the first standardise call
     assert dict(fp_data.data.chunks) == {"time": (2, 2, 2), "lat": (12,), "lon": (12,), "height": (20,)}
+
+
+def test_incompatible_species_for_flux_timeseries():
+    """This function tests if incompatible species values is supplied to standardise"""
+
+    data_path = get_flux_timeseries_datapath(filename="GBR_2023_2021_13042023_170954.xlsx")
+    with pytest.raises(ValueError):
+        standardise_flux_timeseries(
+            filepath=data_path, species="hfc123", source="crf", period="years", continuous=False, store="user"
+        )
+
+
+def test_standardise_flux_timeseries():
+    """This function tests flux_timeseries standardisation function"""
+
+    data_path = get_flux_timeseries_datapath(filename="GBR_2023_2021_13042023_170954.xlsx")
+    flux_results = standardise_flux_timeseries(
+        filepath=data_path, species="ch4", source="crf", period="years", continuous=False, store="user"
+    )
+
+    assert "ch4_crf_UK" in flux_results
```

Comment:
Just a note that if the `region` keyword gets passed to the `clean_string` function (as mentioned above), the key here will include a lowercase "UK" e.g.
```suggestion
    assert "ch4_crf_uk" in flux_results
```